### PR TITLE
Version 1.4.x

### PR DESCRIPTION
**Pull request by psakar**
### Checklist:

* [ ] Have you added a note in the CHANGELOG.md for your change if user-facing?
no
* [ ] Have you added unit tests for your change?
no

Initial python 3.x compatiblity.
Requires configparser library change from https://github.com/project-ncl/pnc-cli/pull/130

### CODE REVIEW

**Comment by thescouser89** [comment]
Adding "future" to our Jenkins nodes. Will retest afterwards

**Comment by thescouser89** [comment]
retest this please

File: pnc_cli/tools/config_utils.py [hunk 1]
```diff
@@ -4,11 +4,11 @@
...
 import string
-from ConfigParser import ConfigParser, NoOptionError, NoSectionError, DuplicateSectionError
+from configparser import ConfigParser, NoOptionError, NoSectionError, DuplicateSectionError
 
-import utils
-from scm_utils import ScmInfo, get_scm_info
-from tasks import Tasks
+# import utils
+# from scm_utils import ScmInfo, get_scm_info
+# from tasks import Tasks
```
Comment (janinko) [inline pnc_cli/tools/config_utils.py]: Don't comment out code, either keep it or remove it.

**Comment by psakar** [comment]
@janiko Please note I have no time for the requested changes as I've indicated via email. May be instead of requesting changes doing them directly would be quicker ?

**Comment by thescouser89** [comment]
I'll try to do the necessary adjustments

File: pnc_cli/tools/config_utils.py (hunk 1 above)
Comment (thescouser89) [reply pnc_cli/tools/config_utils.py]: Good catch! We're actually using all those imports. It should be commented out

**Comment by thescouser89** [comment]
At this point, I think the best course of action to resolve the failure to rebase due to conflicts is to cherry-pick Petr's commit on top of version-1.4.x branch, and apply some changes on top to get the PR more easily rebased. I'll create another PR for it on Thursday hopefully. I'm keeping it open for now so that I remember :D

File: pnc_cli/tools/config_utils.py (hunk 1 above)
Comment (psakar) [reply pnc_cli/tools/config_utils.py]: right. mistake. when you will remove the comment, you must change the import to conform to python 2 *and* python 3 rules at the same time (ATM it works only in python 2 but not in 3) - eg.  "import utils" -> "from . import utils"

**Comment by psakar** [comment]
@thescouser89 Big thanx for taking over !

**Comment by thescouser89** [comment]
PR created here: https://github.com/project-ncl/pnc-cli/pull/134

Will close this PR.

**Pull request closed without merging**